specification/azurefleet/AzureFleet.Management/client.tsp

```diff
@@ -2,5 +2,367 @@ import "./main.tsp";
+@@clientName(DiffDiskPlacement, "ComputeFleetDiffDiskPlacement", "csharp");
+@@clientName(DiffDiskSettings, "ComputeFleetDiffDiskSettings", "csharp");
+@@clientName(DiskControllerTypes, "ComputeFleetDiskControllerTypes", "csharp");
+@@clientName(DiskCreateOptionTypes,
+  "ComputeFleetDiskCreateOptionTypes",
+  "csharp"
+);
+@@clientName(DiskDeleteOptionTypes,
+  "ComputeFleetDiskDeleteOptionTypes",
+  "csharp"
+);
+@@clientName(DomainNameLabelScopeTypes,
+  "ComputeFleetDomainNameLabelScopeTypes",
+  "csharp"
+);
+@@clientName(EncryptionIdentity, "ComputeFleetEncryptionIdentity", "csharp");
+@@clientName(EvictionPolicy, "ComputeFleetEvictionPolicy", "csharp");
+@@clientName(ImageReference, "ComputeFleetImageReference", "csharp");
+@@clientName(InnerError, "ComputeFleetInnerError", "csharp");
+@@clientName(IPVersion, "ComputeFleetIPVersion", "csharp");
+@@clientName(KeyVaultSecretReference,
+  "ComputeFleetKeyVaultSecretReference",
+  "csharp"
+);
+@@clientName(LinuxConfiguration, "ComputeFleetLinuxConfiguration", "csharp");
+@@clientName(LinuxPatchAssessmentMode,
+  "ComputeFleetLinuxPatchAssessmentMode",
+  "csharp"
+);
+@@clientName(LinuxPatchSettings, "ComputeFleetLinuxPatchSettings", "csharp");
+@@clientName(LinuxVMGuestPatchAutomaticByPlatformRebootSetting,
+  "ComputeFleetLinuxVMGuestPatchAutomaticByPlatformRebootSetting",
+  "csharp"
+);
+@@clientName(LinuxVMGuestPatchAutomaticByPlatformSettings,
+  "ComputeFleetLinuxVMGuestPatchAutomaticByPlatformSettings",
+  "csharp"
+);
+@@clientName(LinuxVMGuestPatchMode,
+  "ComputeFleetLinuxVMGuestPatchMode",
+  "csharp"
+);
+@@clientName(NetworkApiVersion, "ComputeFleetNetworkApiVersion", "csharp");
+@@clientName(NetworkInterfaceAuxiliaryMode,
+  "ComputeFleetNetworkInterfaceAuxiliaryMode",
+  "csharp"
+);
+@@clientName(NetworkInterfaceAuxiliarySku,
+  "ComputeFleetNetworkInterfaceAuxiliarySku",
+  "csharp"
+);
+@@clientName(OperatingSystemTypes,
+  "ComputeFleetOperatingSystemTypes",
+  "csharp"
+);
+@@clientName(OSImageNotificationProfile,
+  "ComputeFleetOSImageNotificationProfile",
+  "csharp"
+);
+@@clientName(ProtocolTypes, "ComputeFleetProtocolTypes", "csharp");
+@@clientName(ProvisioningState, "ComputeFleetProvisioningState", "csharp");
+@@clientName(ProxyAgentSettings, "ComputeFleetProxyAgentSettings", "csharp");
+@@clientName(PublicIPAddressSku, "ComputeFleetPublicIPAddressSku", "csharp");
+@@clientName(PublicIPAddressSkuName,
+  "ComputeFleetPublicIPAddressSkuName",
+  "csharp"
+);
+@@clientName(PublicIPAddressSkuTier,
+  "ComputeFleetPublicIPAddressSkuTier",
+  "csharp"
+);
+@@clientName(ScheduledEventsProfile,
+  "ComputeFleetScheduledEventsProfile",
+  "csharp"
+);
+@@clientName(SecurityEncryptionTypes,
+  "ComputeFleetSecurityEncryptionTypes",
+  "csharp"
+);
+@@clientName(SecurityPostureReference,
+  "ComputeFleetSecurityPostureReference",
+  "csharp"
+);
+@@clientName(SecurityProfile, "ComputeFleetSecurityProfile", "csharp");
```

Comment:
Are all these prefixes auto generated or how did we decide where to add ComputeFleet as a prefix?

Reply:
The general rule is, .NET SDK would like to avoid duplicated model naming across different SDK namespaces.
So, if the naming is too generic, we will add the RP prefix for the model to make it more specific.